index.html.slim

```diff
@@ -29,17 +29,13 @@ description: The simple, modern & productive way to build apps and infrastructur
           img.mobile(src=cdn("#{site.base_url}/images/branding/hero_mobile_480x500_eap_01112016.png") alt="Red Hat JBoss EAP 7 Beta now available")
           img.desktop(src=cdn("#{site.base_url}/images/branding/hero_desktop_1175x400_eap_01112016.png") alt="Red Hat JBoss EAP 7 Beta now available")
       .large-24.slide
-        a( href="http://developers.redhat.com/events/judcon/2016/brno/")
-          img.mobile(src=cdn("#{site.base_url}/images/events/hero_mobile_480x500_judcon2016brno.png") alt="Red Hat Developers at JUDCon - Brno")
-          img.desktop(src=cdn("#{site.base_url}/images/events/hero_desktop_1175x400_judcon.png") alt="Red Hat Developers at JUDCon - Brno")
```

Comment:
Never mind, I've merged this anyway as we need this change now. It's not a big deal as this is on the JBD repo.